src/pkg/retention/policy/models.go

```diff
@@ -53,27 +53,24 @@ type Metadata struct {
 
 	// Which scope the policy will be applied to
 	Scope *Scope `json:"scope" valid:"Required"`
-
-	// The max number of rules in a policy
-	Capacity int `json:"cap"`
 }
 
 // Valid Valid
 func (m *Metadata) Valid(v *validation.Validation) {
 	if m.Trigger == nil {
-		_ = v.SetError("Trigger", "Trigger is required")
+		_ = v.SetError("Trigger", "Can not be empty")
```

Comment:
Seems `"Can not be empty"` is an error message, then it should start with lower case letter

Reply:
Here I use upper case is because beego's default error message is using this.

![image](https://user-images.githubusercontent.com/188115/64396730-2b7da280-d091-11e9-89e5-d19cffb743f2.png)

Reply:
ok, let's keep consistent with beego. For our own created error, should be lower case error message.